experiment/autobumper/bumper/bumper.go

```diff
@@ -84,55 +84,55 @@ func (af *fileArrayFlag) Set(value string) error {
+	// The URL where upstream images are located. Only required if Target Version is "upstream" or "upstreamStaging"
 	UpstreamURLBase string `yaml:"upstreamURLBase"`
-	//The config paths to be included in this bump, in which only .yaml files will be considered. By default all files are included.
+	// The config paths to be included in this bump, in which only .yaml files will be considered. By default all files are included.
 	IncludedConfigPaths []string `yaml:"includedConfigPaths"`
-	//The config paths to be excluded in this bump, in which only .yaml files will be considered.
+	// The config paths to be excluded in this bump, in which only .yaml files will be considered.
 	ExcludedConfigPaths []string `yaml:"excludedConfigPaths"`
-	//The extra non-yaml file to be considered in this bump.
+	// The extra non-yaml file to be considered in this bump.
 	ExtraFiles []string `yaml:"extraFiles"`
-	//The target version to bump images version to, which can be one of latest, upstream, upstream-staging and vYYYYMMDD-deadbeef.
+	// The target version to bump images version to, which can be one of latest, upstream, upstream-staging and vYYYYMMDD-deadbeef.
 	TargetVersion string `yaml:"targetVersion"`
-	//The name used in the address when creating remote. Format will be git@github.com:{GitLogin}/{RemoteName}.git
+	// The name used in the address when creating remote. Format will be git@github.com:{GitLogin}/{RemoteName}.git
 	RemoteName string `yaml:"remoteName"`
-	//The name of the branch that will be used when creating the pull request. If unset, defaults to "autobump".
+	// The name of the branch that will be used when creating the pull request. If unset, defaults to "autobump".
 	HeadBranchName string `yaml:"headBranchName"`
-	//List of prefixes that the autobumped is looking for, and other information needed to bump them
+	// List of prefixes that the autobumped is looking for, and other information needed to bump them. Must have at least 1 prefix.
 	Prefixes []Prefix `yaml:"prefixes"`
 }
 
 // Prefix is the information needed for each prefix being bumped.
 type Prefix struct {
-	//Name of the tool being bumped
+	// Name of the tool being bumped
 	Name string `yaml:"name"`
-	//The image prefix that the autobumper should look for
+	// The image prefix that the autobumper should look for
 	Prefix string `yaml:"prefix"`
-	//File that is looked at when bumping to match upstream
+	// File that is looked at to determine current upstream image when bumping to upstream. Required only if targetVersion is "upstream"
 	RefConfigFile string `yaml:"refConfigFile"`
-	//File that is looked at when bumping to match upstreamStaging
+	// File that is looked at to determine current upstream staging image when bumping to upstream staging. Required only if targetVersion is "upstream-staging"
 	StagingRefConfigFile string `yaml:"stagingRefConfigFile"`
-	//Repo used when generating pull request
+	// Repo used when generating pull request
```

Comment:
Source or target repo?

Reply:
Not sure I understand exactly what is meant by source or target repo, but I updated the comment to more explicitly define what repo this field refers to and how it get used.